2-mandatory/3-financial-times.js

```diff
@@ -14,7 +26,34 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let countLowest = 0, lowestTitle = 0, count;
+
+    for (let i = 0; i < allArticleTitles.length; i++)
+    {
+        str = allArticleTitles[i].toString();
+        count = 0;     
+
+        for(j of str)
+        {
+            if (j === ' ')
+            {
```

Comment:
Could this function not be coded a bit more simpler? Hint: Try using the .length property for each element in the allArticleTitles array and keep track of the shortest result.